gammapy/utils/regions.py

```diff
@@ -422,3 +424,46 @@ def region_circle_to_ellipse(region):
         center=region.center, width=region.radius, height=region.radius
     )
     return region_new
+
+
+def extract_bright_star_regions(geom, max_star_mag=6, star_rad=0.3 * u.deg):
+    """
+    Find bright star exclusion regions contained in a given `~gammapy.maps.Geom` and returns a 2D exclusion mask to remove these sources from bacgkround estimation.
+
```

Comment:
You can precise in the docstring which star catalogue is used, and adds a reference towards it.